### PR TITLE
fixed syn offset in solutions for exercises 11 through 14.

### PR DESCRIPTION
**Pull request by rshpeley**
Further to #243, #256 and #260 this update fixes example solutions in exercises 11 through 14.

The issue that prevented replicating the exercise 7 plot for exercise 11 was due to the improper indexing of syn. Changing,
```python
v[step] = v[step - 1] + (dt / tau) * (el - v[step - 1] + r * syn[step])
``` 
to
```python
v[step] = v[step - 1] + (dt / tau) * (el - v[step - 1] + r * syn[step-1])
```
corrected the plot to the same as in exercise 7. Opening the two plots in new tabs confirms this when switching between them.

However, the problem with indexing from the `1` position caused problems in exercises 12 through 14 and it was simpler to change the loop to index from `0`. For example, exercise 12 is simplified from,

```python
# loop for step_end values of syn
for step, i in enumerate(syn):

  # skip first iteration
  if step==0:
    continue

  v[step] = v[step - 1] + (dt / tau) * (el - v[step - 1] + r*i)
```
to
```python
# loop for step_end -1 values of syn
for step, i in enumerate(syn[:-1]):
  v[step + 1] = v[step] + (el - v[step] + r * i) * dt / tau
```
while still correcting for the syn offset error.

### CODE REVIEW

File: tutorials/W0D1_PythonWorkshop1/W0D1_Tutorial1.ipynb [hunk 1]
```diff
@@ -2235,16 +2227,16 @@
...
     "  plt.plot(t_range, v_mean, 'C0', alpha=0.8, label='mean')\n",
-    "  plt.plot(t_range, v_mean+v_std, 'C7', alpha=0.8)\n",
-    "  plt.plot(t_range, v_mean-v_std, 'C7', alpha=0.8, label='mean $\\pm$ std')\n",
+    "  plt.plot(t_range, v_mean + v_std, 'C7', alpha=0.8)\n",
+    "  plt.plot(t_range, v_mean - v_std, 'C7', alpha=0.8, label='mean $\\pm$ std')\n",
     "\n",
     "  # # alternative for filling plot between v_mean-+v_std\n",
     "  # plt.fill_between(t_range, v_mean-v_std, v_mean+v_std, color='C0', alpha=0.6)\n",
     "  \n",
-    "  plt.legend()\n",
```
Comment (durmusgulbahar) [inline tutorials/W0D1_PythonWorkshop1/W0D1_Tutorial1.ipynb:2247]: print("im am new sorry")

Comment (rshpeley) [reply tutorials/W0D1_PythonWorkshop1/W0D1_Tutorial1.ipynb:2247]: ??

**Comment by mwaskom** [comment]
Hi @rshpeley , per the [contributing guidelines](https://github.com/NeuromatchAcademy/course-content/blob/master/.github/contributing.md), we would prefer that all problems with the content be reported as issues rather than PRs so that we can involve the original content creators and keep the PR queue clear for content finalization. Thank you for helping make Neuromatch content better and for understanding — I know it's frustrating to put in the time submitting a PR and then having it closed.

**Comment by rshpeley** [comment]
Good to see you've created guidelines now through #340, although you may want to put the guidelines doc in a more visible location. Unfortunately, as a participating student I need to focus my time on the course and project rather than creating new issues. Perhaps you can have someone review submitted and cancelled PRs at a later date.
--

**Pull request closed without merging**